infrastructure/lib/atat-web-api-stack.ts

```diff
@@ -537,6 +540,9 @@ export class AtatWebApiStack extends cdk.Stack {
       handlerPath: this.determineApiHandlerPath(operationId, handlerFolder),
       database: this.database,
       ormLayer: this.ormLayer,
+      functionPropsOverride: {
+        timeout: cdk.Duration.seconds(10),
+      },
```

Comment:
Fair but we do need to debug this. We can't actually support 10s for each query; the UI will crawl

Reply:
Usually I see it taking about 6 seconds initially.

Reply:
Well, we need the timeout to be big enough for a cold start right? So technically the first query could be unacceptably slow as long as subsequent ones are fine. Granted, we'll have to pursue a warming strategy if this is the case.